src/phoenix/experimental/evals/templates/default_templates.py

```diff
@@ -0,0 +1,38 @@
+RAG_RELEVANCY_PROMPT_TEMPLATE_STR = """
+You are comparing a reference text to a question and trying to determine if the reference text
+contains information relevant to answering the question. Here is the data:
+    [BEGIN DATA]
+    ************
+    [Question]: {query}
+    ************
+    [Reference text]: {reference}
+    [END DATA]
+
+Compare the Question above to the Reference text. You must determine whether the Reference text
+contains information that can answer the Question. Please focus on whether the very specific
+question can be answered by the information in the Reference text.
+Your response must be single word, either "relevant" or "irrelevant",
+and should not contain any text or characters aside from that word.
+"irrelevant" means that the reference text does not contain an answer to the Question.
+"relevant" means the reference text contains an answer to the Question.
+"""
+
+HALLUCINATION_PROMPT_TEMPLATE_STR = """
+In this task, you will be presented with a query, a reference text and an answer. The answer is
+generated to the question based on the reference text. The answer may contain false information, you
+must use the reference text to determine if the answer to the question contains false information,
+if the answer is a hallucination of facts. Your objective is to determine whether the reference text
+contains factual information and is not a hallucination. A 'hallucination' in this context refers to
+an answer that is not based on the reference text or assumes information that is not available in
+the reference text. Your response should be a single word: either "factual" or "hallucinated", and
+it should not include any other text or characters. "hallucinated" indicates that the answer
+provides factually inaccurate information to the query based on the reference text. "factual"
+indicates that the answer to the question is correct relative to the reference text, and does not
+contain made up information. Please read the query and reference text carefully before determining
+your response.
+
+    # Reference text: {passage_text}
```

Comment:
why not keep the variable consistent as reference?

Reply:
Addressed in [586b33b](https://github.com/Arize-ai/phoenix/pull/1301/commits/586b33b60083f23fb69f165a956e040cfec34099)